### PR TITLE
Add planned enhancement for corrupt addressbook.json, move planned enhancements to DG

### PR DESCRIPTION
**Pull request by conradsoon**

### CODE REVIEW

**Comment by conradsoon** [comment]
Resolves #226 #225

File: docs/UserGuide.md
Comment (kokrui) [inline docs/UserGuide.md]: could you add the specific fix we'll implement? e.g. backing-up the old addressbook.json file

**Review by kokrui — APPROVED** [review]
Mostly LGTM, except 1 thing you should add before you merge

**Review by kokrui — CHANGES_REQUESTED** [review]
oops move it to DG please my bad for not catching first time

File: docs/UserGuide.md
Comment (kokrui) [inline docs/UserGuide.md]: could you remove this diff? since the section will be removed once Calvin's currently-approved PR is merged anyways (and this diff would cause merge confclit)

**Review by kokrui — APPROVED** [review]
LGTM! 1 nit to fix if ur free

**Pull request merged**
